docs/ii-spec.mdx

```diff
@@ -293,34 +295,46 @@ In order to allow Internet Identity to read the path `/.well-known/ii-alternativ
 
 This section describes the interface that the backend canister provides.
 
-This interface is currently only used by its own frontend. This tight coupling means that this interface may change, even in incompatible ways. We therefore do not have to apply Candid best practices for backward-compatibility (such as using records for arguments and results).
+Note that the interface is split into 4 categories:
```

Comment:
Why did you remove the warning paragraph of breaking changes?

> This interface is currently only used by its own frontend. This tight coupling means that this interface may change, even in incompatible ways. We therefore do not have to apply Candid best practices for backward-compatibility (such as using records for arguments and results)

Reply:
Well, because it is not true. There are at least 2 other products that use the II back-end in some capacity.

Reply:
Oh, I didn't know! Ok, in that case, we do need to keep backwards compatibility. Good to know!